app/index.html

```diff
@@ -15,16 +15,22 @@
         <link rel="stylesheet" href="css/all.css"/>
         <!-- /build -->
 
-         <!-- Facebook  -->
-        <meta name="og:site_name" content="SOON_" />
-        <meta name="og:description" content="We design and build digital products, services and brands." />
-
-        <!-- Twitter  -->
-        <meta name="twitter:card" content="summary" />
-        <meta name="twitter:site" content="@soonlondon" />
-        <meta name="twitter:title" content="SOON_" />
-        <meta name="twitter:description" content="We design and build digital products, services and brands." />
-        <meta name="twitter:url" content="http://thisissoon.com" />
+        <!-- Twitter Card data -->
+        <meta name="twitter:card" content="summary">
+        <meta name="twitter:url" content="{{ socialMeta.url || 'http://www.thisissoon.com' }}">
+        <meta name="twitter:site" content="@soonlondon">
+        <meta name="twitter:title" content="{{ socialMeta.title || 'SOON_' }}">
+        <meta name="twitter:description" content="{{ socialMeta.description }}">
+        <meta name="twitter:creator" content="@soonlondon">
+        <meta name="twitter:image:src" content="{{ socialMeta.image }}">
+
+        <!-- Open Graph data -->
+        <meta property="og:title" content="{{ socialMeta.title || 'SOON_' }}" />
+        <meta property="og:type" content="website" />
+        <meta property="og:url" content="{{ socialMeta.url || 'http://www.thisissoon.com' }}"/>
+        <meta property="og:image" content="{{ socialMeta.image }}" />
+        <meta property="og:description" content="{{ socialMeta.description }}" />
+        <meta property="og:site_name" content="SOON_" />
 
```

Comment:
@jamesjwarren this won't work as Facebook will just read this html file without processing it so all the values in content would not get rendered.